Automated Changelog Entry for 0.4.0 on main

martinRenou opened the pull request:
Automated Changelog Entry for 0.4.0 on main
```
Python version: 0.4.0
npm version: @voila-dashboards/voila-root: 0.1.0
npm workspace versions:
@voila-dashboards/jupyterlab-preview: 2.2.0
@voila-dashboards/voila: 0.4.0
```

After merging this PR run the "Full Release" Workflow on your fork of `jupyter_releaser` with the following inputs
| Input  | Value |
| ------------- | ------------- |
| Draft Release | https://github.com/voila-dashboards/voila/releases/tag/untagged-1f3a0e9981b4d84898e8  |
| Since | @voila-dashboards/jupyterlab-preview@2.2.0-rc.0 |

jtpio commented:
Probably the workflow could have been run with "Since last stable" to automatically include all the changes since the last `0.3.x` release.

martinRenou commented:
> Probably the workflow could have been run with "Since last stable" to automatically include all the changes since the last 0.3.x release.

Thanks! I was wondering how this should have been done. I'll run the bot again.
